### DISCUSSION TITLE
Question: Example architecture - hardware used

### DISCUSSION THREAD

**Discussion by aottVESartex** [joyja/tentacle-plc#16] [Q&A]
This looks like a very interesting project and I will definitely test this out when I have the time.

I have a few questions regarding your example architecture below:
![image](https://user-images.githubusercontent.com/91678829/166976173-3b8205d7-3225-498e-b568-09636c831a07.png)

- Do you have examples of the hardware used to test the Tentacle PLC with?
  - Specifically, the remote IO?
  - Based on your company's site, I presume the Linux IPC you use is the Fitlet?

Thanks!

**Comment by joyja** [comment]
Hi @aottVESartex thanks! Yeah I typically use Ubuntu on Compulab Fitlet 2. It's a nice platform at a good price. For IO, I've been working with Moxa ioThinx 4510 and am currently trying out ODOT. Probably should post a more detailed layout of my lab setup in the Tentacle PLC docs at some point.

**Comment by aott33** [comment]
Thanks for the response, @joyja. I asked the question on my work GitHub account. Meant to ask it on my personal GitHub account.

Cool, I tested out the [EZvpn IOX1](https://iocondocs.ezvpn.online/hardware/iox1/) which uses the Fitlet 2. I recently ordered the Fitlet 3 and waiting for that to arrive so that I can do some testing.

Nice, I have been seeing quite a bit of ODOT remote IO referenced on LinkedIn. Let me know how the the ODOT performs. I am interested testing it out as well. 

Some additional content on your lab setup would be greatly appreciated. Thanks!
